### DISCUSSION TITLE
doris编译失败!!(cloudera-public)

### DISCUSSION THREAD

**Discussion by issue-K** [apache/doris#34406] [A - General / Q&A]
Downloading from cloudera-public: https://repository.cloudera.com/artifactory/public/net/sourceforge/czt/dev/cup-maven-plugin/1.6-cdh/cup-maven-plugin-1.6-cdh.jar
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for fe-common 1.2-SNAPSHOT:
[INFO]
[INFO] fe-common .......................................... SUCCESS [ 44.942 s]
[INFO] spark-dpp .......................................... SUCCESS [ 37.861 s]
[INFO] fe-core ............................................ FAILURE [  5.077 s]
[INFO] hive-udf ........................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:28 min
[INFO] Finished at: 2024-05-05T17:52:35+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Plugin net.sourceforge.czt.dev:cup-maven-plugin:1.6-cdh or one of its dependencies could not be resolved: The following artifacts could not be resolved: net.sourceforge.czt.dev:cup-maven-plugin:jar:1.6-cdh (present, but unavailable): Could not find artifact net.sourceforge.czt.dev:cup-maven-plugin:jar:1.6-cdh in central (https://repo.maven.apache.org/maven2) -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginResolutionException

There seems to be a problem with cloudera-public configured in fe/pom.xml. How do I fix it?
看起来是fe/pom.xml中配置的cloudera-public有问题，我应该如何解决？

**Comment by yagagagaga** [comment]
You can upgrade to the latest version of Doris. The POM has updated.

**Reply by issue-K** [reply]
I am using the latest version of release-2.0.1.1. Isn't that right? O O

**Reply by yagagagaga** [reply]
Check if the POM contains the following code
```xml
<!-- for cup-maven-plugin -->
                <!-- https://docs.cloudera.com/documentation/enterprise/release-notes/topics/cdh_vd_cdh5_maven_repo.html -->
                <pluginRepository>
                    <id>cloudera</id>
                    <url>https://repository.cloudera.com/repository/libs-release-local/</url>
                </pluginRepository>
```

**Reply by issue-K** [reply]
The compilation succeeded after changing the branch. Thank you so much~
